### ISSUE TITLE
Use validation data for inference

### ISSUE THREAD

**Issue by LHJ97** [MIC-DKFZ/nnUNet#596]
Hi Fabian,

I want to test nnUNet on my own dataset without running all the 5 folds. Is there a way to extract the validation data, for example trained on fold 0 train data and run inference on fold 0 validation data?

Another question on plan and preprocess the dataset. My dataset is around 56Gb. The preprocessed data saved in nnUNet_preprocessed(150Gb) and the cropped data (60Gb) saved in nnUNet_raw_data_base take way too much space on my disk. Is it normal to get such a large cropped and preprocess folder?

Thanks a lot for your help!

**Comment by FabianIsensee** [comment]
Hi,
yes it is normal that the datasets are this large. It would require a lot of work to change that.
Note that when you are done with training and do not need the dataset in a while you can delete all the npy files in the nnUNet_preprocessed folder.

> I want to test nnUNet on my own dataset without running all the 5 folds. Is there a way to extract the validation data, for example trained on fold 0 train data and run inference on fold 0 validation data?

Just let fold0 train and then have a look at the training output directory. There will be a validation_raw folder with a summary.json in it.
Best,
Fabian

**Comment by LHJ97** [comment]
Thanks for your reply! I started the training on the GPU cluster with a time limit of 72 hours. The GPU used was V100 32Gb. However, the training was not finished after 72 hours and my job was aborted. You stated that the training should finish in two days, is it normal that my training only reaches 330 epochs after 72 hours? One epoch took around 12 minutes. 

I can't find the validation_raw folder in the RESULTS_FOLDER. What I have is a debug.json, some models files, a progress.png and a training log file. Is this due to the incomplete training?

**Comment by FabianIsensee** [comment]
Please read this: https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/expected_epoch_times.md

And run the benchmarks :-)

**Comment by alexgo1** [comment]
> Just let fold0 train and then have a look at the training output directory. There will be a validation_raw folder with a summary.json in it.
> Best,
> Fabian

I wonder, what's the best way to achieve that if I don't want to repeat training?
An example use case is if I trained a model, but now I want to plug in a different evaluation function.
I still want to run it only on the validation cases of the fold that I used.

I assume this can be achieved by creating a new directory which is a subset of the raw dataset that contains only the specific cases used for validation. But I'm wondering if there's a simpler way which doesn't require to copy case files.
